content/rule_book/roles_and_responsibilities.md

```diff
@@ -2,68 +2,96 @@
-        - Once the quizzer has completed all information in the answer, the quizmaster will prompt the quizzer to identify the specific pronoun, even if the quizzer has not yet completed all information in the question.
-    - On a Chapter-Verse Reference Question or Chapter-Only Reference Question, if the quizzer has given all the information in both the question and the answer and the complete question was not read, the quizmaster will ask, "What is your question?"
-    - On a Quote/Finish type question, once the quizzer has quoted to the end of the required verse or verses, if the quote is not yet word-perfect, the quizmaster will say, "Again." This will occur as often as needed until the quizzer quotes the verse perfectly or the 30-second time limit expires.
-        - The quizmaster is not allowed to say "again" until the quizzer has come to the end of the verse, each time through, as this would give the quizzer information about where their mistake was made.
-    - Once all of the information in both the question and answer has been given, but something needs to be fixed, the quizmaster will say "Again".
-    - On a Situation type question, when the quizzer has completed the Quotation the quizmaster will say, "Quote is complete."
-    - On an INT or MA Question, if all of the information in the answer text has been given, but the quizzer is missing information in the question text, the quizmaster will say "More".
-    - On a Situation question, because the quotation is part of the question, the quizmaster will say "More" when required information from the quotation is missing.
-    - On a Q/Q2V, CVR/MACVR, CR/MACR question, if either the chapter or verse number were not completely finished when the quizzer jumped, the quizmaster will say "More".
-    - On a Reference type question, if the quizzer gives a reference question that is different than the written question, the quizmaster and answer judge must ensure that the reference question given is the exact same type as the written question.
-    - Consult with the answer judge(s) at any time they feel is necessary, provided it does not infringe upon the 30 seconds allotted the quizzer.
-    - If an explanation of a ruling is requested by a captain, the quizmaster will ask if any team wants to challenge before the explanation is given.
+- Rule immediately upon an obviously correct or incorrect answer if there is no indication from the answer judge to the contrary
+
+When a quizzer's response contains a required completion of the question and that completion be different than the written question, the quizmaster will discuss it with the answer judge. If the quizzer gives a reference question that is different than the written question, the quizmaster and answer judge must ensure that the reference question given is the exact same type as the written question.
```

Comment:
"that completion _is_ different than the written question"

Reply:
It's "talk like a pirate" rule book, dude. :) 6f5280d